components/typography/styles.js

```diff
@@ -129,10 +132,10 @@ export const _generateBodySmallStyles = (selector) => {
 			max-height: 5rem;
 		}
 		@media (max-width: 615px) {
-			${selector} {
+			${unsafeCSS(!inputStyleTweaksEnabled ? `${selector} {
 				font-size: 0.6rem;
 				line-height: 0.9rem;
-			}
+			}` : '')}
```

Comment:
Design decided that this responsive rule to reduce font size isn't needed anymore.